tests/keras/test_custom_op.py

```diff
@@ -0,0 +1,57 @@
+import sys
+
+import onnxmltools
+import numpy as np
+import unittest
+import keras
+
+from onnxmltools.convert.common.base import *
+from keras import backend as K
+from keras.layers import *
+
+
+class ScaledTanh(keras.layers.Layer):
+    def __init__(self, alpha=1.0, beta=1.0, **kwargs):
+        super(ScaledTanh, self).__init__(**kwargs)
+        self.alpha = alpha
+        self.beta = beta
+
+    def build(self, input_shape):
+        super(ScaledTanh, self).build(input_shape)
+
+    def call(self, x):
+        return self.alpha * K.tanh(self.beta * x)
+
+    def compute_output_shape(self, input_shape):
+        return input_shape
+
+if sys.version_info[0] < 3:
+    def custom_activation(scope, operator, container):
```

Comment:
Could `custom_activation`s be an isolated functions? I guess defining them inside this class force you to create base.py.

Reply:
it's a function. And the intent is because of if statement.